docs/docs/contribute/software/add-new-metric-provider.md

```diff
@@ -0,0 +1,123 @@
+---
+comments: true
+---
+
+# Add a metrics provider endpoint
+
```

Comment:
```suggestion
The
[KeptnMetric](../../../guides/evaluatemetrics.md)
feature works with almost any data platform
but Keptn requires that a metrics provider be defined
for any data platform it uses as a data source.
```
You need to provide the reader with just a bit of context for this page.  I suggested some prose but feel free to alter it.  And check that I coded that xref correctly since I can't test it from my suggestion.